packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/utils/getActionIconColorOrThrow.ts

```diff
@@ -22,8 +22,10 @@ export const getActionIconColorOrThrow = ({
     case 'FORM':
       return theme.color.orange;
     case 'ITERATOR':
-    case 'FILTER':
     case 'EMPTY':
+    case 'DELAY':
+      return theme.color.green60;
```

Comment:
<img width="347" height="131" alt="Image" src="https://github.com/user-attachments/assets/0fb0be0a-e8e8-4ad8-8470-eae4af88c6de" /> This looks odd? Please refer to Figma or check with @Bonapara